bellboy/utils/address_book.py

```diff
@@ -0,0 +1,35 @@
+import logging
+from typing import Dict
+
+from thespian.actors import ActorAddress
+
+from src.utils.constants import ActorNames
+
+
+class AddressBook:
```

Comment:
This class can be nuked, it isn't used and doesn't work correctly anyhow.

Reply:
alright, we can decide later how we want to globally map addresses--> globalNames. its the multiprocess system making it difficult